Examples/Image/Classification/ResNet/Python/TrainResNet_CIFAR10.py

```diff
@@ -7,18 +7,48 @@
 from __future__ import print_function
 import os
 import argparse
+import code
```

Comment:
This file should not be modified as it may cause some other test changes. Only notebook is needed.